### PR TITLE
feat: add story media form

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#1672

Migrate inputs:
- VideoInput
- Radio

Add:
- story media form

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-components/src/components/inputs/new/RadioGroup/RadioGroup.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const Template: ComponentStory<typeof RadioGroup> = args => {
+  const selectedValue = 'video';
+
+  return (
+    <RadioGroup sx={{ maxWidth: 500 }} {...args}>
+      <>
+        <Radio
+          name={'radio-story'}
+          label={'Add a video'}
+          labelOptionalText={RADIO_PREFERABLE}
```
Comment (blushi) [inline web-components/src/components/inputs/new/RadioGroup/RadioGroup.stories.tsx:22]: Maybe best to keep the same prop name as the existing one on `FieldFormControl` (that uses `ControlledFormLabel`)? It uses `optional` as prop name, we could use one or the other, I don't have a preference but let's stay consistent.

**Review by blushi — COMMENTED** [review]
you also need to add `regen:storyMedia` to the list of `UNANCHORED_PROJECT_KEYS`

File: web-components/src/components/inputs/new/RadioGroup/RadioGroup.stories.tsx (hunk 1 above)
Comment (flagrede) [reply web-components/src/components/inputs/new/RadioGroup/RadioGroup.stories.tsx:22]: Fair point, let's stay with `optional` then.

Comment (blushi) [reply web-components/src/components/inputs/new/RadioGroup/RadioGroup.stories.tsx:22]: with the `optional` props you can also put custom text

Comment (blushi) [reply web-components/src/components/inputs/new/RadioGroup/RadioGroup.stories.tsx:22]: see https://github.com/regen-network/regen-web/blob/9252815680e713ba6fd099d5a1e03ddc07e90e6f/web-components/src/components/form/ControlledFormLabel.tsx#L21

**Review by blushi — APPROVED** [review]

**Pull request merged**